### ISSUE TITLE
Drop Custom Font preference

### ISSUE THREAD

**Issue by cassidyjames** [elementary/scratch#129]
Scratch uses whatever the system-wide monospace font is. I don't really see the value in setting a separate monospace font just for a single app.

**Comment by danrabbit** [comment]
I agree, if you really want to change your monospace font you probably want to change it for all apps

**Comment by OttoRobba** [comment]
Counter example for this are monospaced fonts with ligatures - like Fira Code and Monoid - do they work properly with pantheon-terminal? I know that gnome-terminal is quite buggy with them. So I use one type of font in the editor and another in the system. Can anyone test with ligature fonts?

**Comment by cassidyjames** [comment]
As far as I know (and last I checked), GTKSourceView doesn't support ligature fonts, either. Ideally both Terminal and Code would support them with their respective VTE and Source View widgets.

**Comment by OttoRobba** [comment]
GTKSourceView does support them - tested both with Scratch and Gedit.

**Comment by donadigo** [comment]
I disagree, system font != code font. People may have a favourite font just for coding (since Scratch is now only meant for coding).

**Comment by stsdc** [comment]
I use FiraCode only in text editors, because of ligatures.

**Comment by cassidyjames** [comment]
That's a fair point; when I use Atom, I also use Fira Code for its awesome ligatures. If this is a common enough thing, I could see keeping a font preference. I'd also be curious if it'd make more sense to enable custom fonts with a plugin, or if that's just overcomplicating things.

**Comment by OttoRobba** [comment]
Hmm that does seem to overcomplicate things a bit.

Tested with pantheon-terminal, it does not handle ligatures correctly.

What I typed was:
```bash
echo "--> this seems wrong <--"
```

But output was:
![captura de tela de 2017-08-18 20-27-01](https://user-images.githubusercontent.com/597287/29481039-a85843d2-8453-11e7-8188-de3308326930.png)

I'm ok with removing the font option but we would need to fix how the terminal handles ligatures (and check if they are buggy anywhere else). Seems easier to just let it be.

**Comment by OttoRobba** [comment]
Addendum - everyone seems to really like Fira Code. Maybe make that the Code default?

**Comment by yeboahnanaosei** [comment]
Guys, I don't think you should drop the custom font preference. Roboto Mono is my system wide monospace font but I however find myself switching between Inconsolata and Roboto Mono for my text editors (I use atom and scratch). It's always nice to set the font in your editor independently of the system wide settings.

Don't take this option away from scratch. :worried:

**Comment by rosavi** [comment]
Would changing the Gtk.FontButton to a SpinButton that sets the font size be an acceptable fix? I'm having a go at this. What's the best place to ask questions?

**Comment by cassidyjames** [comment]
@rosavi I think if we're keeping a preference around, we might as well use a standard font selection. Someone might install a font that we don't know about, so I don't want to hardcode a list.
